### ISSUE TITLE
Calling get_cutout() results in a server error page

### ISSUE THREAD

**Issue by ahota** [idies/pyJHTDB#17]
Hello,

I'm trying to get cuts from the coarse isotropic dataset. I tried calling `get_cutout(filename='/path/to/file', auth_token='my.auth.token')`, letting most of the parameters stay as defaults. The URL it prints out looks like the following (shown here with the default auth token):   `http://turbulence.pha.jhu.edu/cutout/download.aspx/edu.jhu.pha.turbulence.testing-201302/isotropic1024coarse/u/0,1/0,16/0,16/0,16/`

It then gives me an error that it was unable to open the downloaded HDF5 file (line 63 of cutout.py). I opened it in less and saw it was an HTML page. If I load the URL in a browser, it shows a server-side error with the following information:

```
Server Error in '/cutout' Application.

Could not load file or assembly 'HDF5DotNet.DLL' or one of its dependencies. The specified module could not be found.

Description: An unhandled exception occurred during the execution of the current web request. Please review the stack trace for more information about the error and where it originated in the code. 

Exception Details: System.IO.FileNotFoundException: Could not load file or assembly 'HDF5DotNet.DLL' or one of its dependencies. The specified module could not be found.

Source Error: 

An unhandled exception was generated during the execution of the current web request. Information regarding the origin and location of the exception can be identified using the exception stack trace below.

Stack Trace: 


[FileNotFoundException: Could not load file or assembly 'HDF5DotNet.DLL' or one of its dependencies. The specified module could not be found.]
   System.Reflection.RuntimeAssembly._nLoad(AssemblyName fileName, String codeBase, Evidence assemblySecurity, RuntimeAssembly locationHint, StackCrawlMark& stackMark, IntPtr pPrivHostBinder, Boolean throwOnFileNotFound, Boolean forIntrospection, Boolean suppressSecurityChecks) +0
   System.Reflection.RuntimeAssembly.InternalLoadAssemblyName(AssemblyName assemblyRef, Evidence assemblySecurity, RuntimeAssembly reqAssembly, StackCrawlMark& stackMark, IntPtr pPrivHostBinder, Boolean throwOnFileNotFound, Boolean forIntrospection, Boolean suppressSecurityChecks) +457
   System.Reflection.RuntimeAssembly.InternalLoad(String assemblyString, Evidence assemblySecurity, StackCrawlMark& stackMark, IntPtr pPrivHostBinder, Boolean forIntrospection) +110
   System.Reflection.RuntimeAssembly.InternalLoad(String assemblyString, Evidence assemblySecurity, StackCrawlMark& stackMark, Boolean forIntrospection) +22
   System.Reflection.Assembly.Load(String assemblyString) +34
   System.Web.Configuration.CompilationSection.LoadAssemblyHelper(String assemblyName, Boolean starDirective) +48

[ConfigurationErrorsException: Could not load file or assembly 'HDF5DotNet.DLL' or one of its dependencies. The specified module could not be found.]
   System.Web.Configuration.CompilationSection.LoadAssemblyHelper(String assemblyName, Boolean starDirective) +729
   System.Web.Configuration.CompilationSection.LoadAllAssembliesFromAppDomainBinDirectory() +247
   System.Web.Configuration.CompilationSection.LoadAssembly(AssemblyInfo ai) +157
   System.Web.Compilation.BuildManager.GetReferencedAssemblies(CompilationSection compConfig) +226
   System.Web.Compilation.BuildManager.GetPreStartInitMethodsFromReferencedAssemblies() +73
   System.Web.Compilation.BuildManager.CallPreStartInitMethods(String preStartInitListPath, Boolean& isRefAssemblyLoaded) +321
   System.Web.Compilation.BuildManager.ExecutePreAppStart() +170
   System.Web.Hosting.HostingEnvironment.Initialize(ApplicationManager appManager, IApplicationHost appHost, IConfigMapPathFactory configMapPathFactory, HostingEnvironmentParameters hostingParameters, PolicyLevel policyLevel, Exception appDomainCreationException) +878

[HttpException (0x80004005): Could not load file or assembly 'HDF5DotNet.DLL' or one of its dependencies. The specified module could not be found.]
   System.Web.HttpRuntime.FirstRequestInit(HttpContext context) +525
   System.Web.HttpRuntime.EnsureFirstRequestInit(HttpContext context) +124
   System.Web.HttpRuntime.ProcessRequestNotificationPrivate(IIS7WorkerRequest wr, HttpContext context) +700

Version Information: Microsoft .NET Framework Version:4.0.30319; ASP.NET Version:4.7.2053.0
```

Did I call `get_cutout()` incorrectly? Ultimately what I want to do is grab a large section of the data, e.g. 100 time steps of the full 1024^3 extent of the dataset.

**Comment by sshamilton** [comment]
That is the old cutout service link.  If that is in the code, we need to
update the link.

**Comment by ahota** [comment]
I see. Line 42 of cutout.py is where the link is built based on the parameters.

**Comment by chichilalescu** [comment]
@sshamilton , can you please confirm that "http://dsp033.pha.jhu.edu/jhtdb/getcutout/" is the correct thing to put in front of the authorization token etc? I'm a bit surprised about the specificity of "dsp033".

**Comment by sshamilton** [comment]
Chichi,
Yes that is the correct link.  I was trying to get it moved to a production
webserver cluster, but I wasn't able to before I left JHU.
The server is pretty solid though--it is way better than the old cutout
service.
Stephen

**Comment by chichilalescu** [comment]
Fixed in commit 8b6ca19d75fb84fba36fc3b1238c314b8664f2b8
